src/lib/optimizer/strategy/subquery_to_join_rule.cpp

```diff
@@ -199,6 +199,26 @@ std::pair<SubqueryToJoinRule::PredicatePullUpResult, bool> pull_up_correlated_pr
   return {result, false};
 }
 
+void push_arithmetic_expression_into_subquery(std::shared_ptr<BinaryPredicateExpression>& predicate_expression,
+                                              const std::shared_ptr<ArithmeticExpression>& arithmetic_expression,
+                                              std::shared_ptr<LQPSubqueryExpression>& subquery_expression,
+                                              size_t arithmetic_expression_position, bool subquery_expression_is_left) {
```

Comment:
rename `arithmetic_expression_position` to `arithmetic_expression_argument_idx`?